BUILDING.md

```diff
@@ -0,0 +1,57 @@
+Building sailjail
+=================
+Sailjail uses [Meson build system](https://mesonbuild.com/). To build it
+use the following commands:
+
+    meson setup build
+    meson compile -C build
+
+See [the next section](#useful-setup-options) for more tips.
+
+Useful setup options
+--------------------
+Some useful options for `meson setup`:
+
+| option                   | explanation                             |
+| -----------------------: | :-------------------------------------  |
+| -Db\_coverage=true       | Build with support for coverage reports |
+| -Dlibdbusaccess=disabled | Build without libdbusaccess support     |
+
+Running unit tests
+------------------
+Unit tests can be run with meson:
+
+    meson test -C build
+
+After that you can also generate a coverage report if you built with
+coverage reports support:
+
+    ninja coverage -C build
```

Comment:
This also caused me trouble due to a missing `gconv` (I eventually got it to pip install in `sb2` and then everything worked fine, but is there a better way?).

Reply:
I built and run all unit tests outside our SDK. I'll just leave it for QA devices to run the QA versions of the tests on our platform.

Reply:
Understood. I had trouble building outside the sdk due to missing dependencies, but I didn't try the `-D libdbusaccess=disabled` flag; thanks for explaining that to me.